pkg/kubelet/server.go

```diff
@@ -64,9 +68,9 @@ type TLSOptions struct {
 }
 
 // ListenAndServeKubeletServer initializes a server to respond to HTTP network requests on the Kubelet.
-func ListenAndServeKubeletServer(host HostInterface, address net.IP, port uint, tlsOptions *TLSOptions, enableDebuggingHandlers bool) {
+func ListenAndServeKubeletServer(host HostInterface, auth AuthInterface, address net.IP, port uint, tlsOptions *TLSOptions, enableDebuggingHandlers bool) {
 	glog.Infof("Starting to listen on %s:%d", address, port)
-	handler := NewServer(host, enableDebuggingHandlers)
+	handler := NewServer(host, auth, enableDebuggingHandlers, enableDebuggingHandlers)
```

Comment:
metrics were being registered in two places: in the readonly server setup, and within InstallDebuggingHandlers() in the read/write server setup... this is the first step in unifying registration without rippling too many changes